src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -442,6 +442,9 @@ ClientRequest::do_process(
       co_return;
     }
   }
+
+  co_await maybe_inject_delay();
```

Comment:
@cyx1231st, this should be helpful for debugging. Are you ok with introducing this step?

Reply:
Ack, I think so.